Sources/Shared/UI/Dashboard/DashboardViewController.m

```diff
@@ -60,34 +60,6 @@ - (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibB
     return self;
 }
 
-- (void)dealloc
-{
-    [_arrDashboard release];
-	_arrDashboard = nil;
-    
-    _tblGadgets = nil;
-    
-    [_refreshHeaderView release];
-    _refreshHeaderView = nil;
-    
-    [_dateOfLastUpdate release];
-    _dateOfLastUpdate = nil;
-    
-    [_refreshHeaderView release];
-    _refreshHeaderView = nil;
-    
-    _dashboardProxy.delegate = nil;
-    [_dashboardProxy release];
-    _dashboardProxy = nil;
-    
-    [_errorForRetrievingDashboard release];
-    _errorForRetrievingDashboard = nil;
-    
-    [[NSNotificationCenter defaultCenter] removeObserver:self];
```

Comment:
You must keep the `dealloc` method to remove the observer. Do not call `super dealloc` or `release` anything though as the compiler will add those statements.